documentation/decisions/adr/2023-10-16-email-marketing.md

```diff
@@ -0,0 +1,215 @@
+
+## Options Considered
+
+
+- [Sendy](https://sendy.co/) 
+- [Salesforce Marketing](https://www.salesforce.com/products/engagement-marketing/) - 
+- [MailChimp](https://mailchimp.com/)
+- [Hubspot](https://www.hubspot.com/)
+- [Mautic](https://www.mautic.org/)
+- [Adobe Campaign](https://business.adobe.com/products/campaign/adobe-campaign.html)
+
+## Decision Outcome <!-- REQUIRED -->
+
+The suggested approach is to utilize Sendy and create a campaign for Simpler Grants.gov within the existing main account, leveraging existing knowledge and allowing for faster use. This allows for a faster initiation of the process. One of the most important criteria given by users of the tool, the Grants.gov Communications team, is the need to be able to send an email to the current 1M current subscribers quickly which Sendy cannot do. There are other limitations of this solution so we recommend that we select a back-up option email marketing solution for our long-term needs. 
+
+
+
+### Open Questions <!-- OPTIONAL -->
+
+1. Can we transfer the Simpler Grants.gov email list to a different tool in the future?
+
+   TODO
+
+
+
+### Back-up Options
+
+Alternatively, as a backup solution, Mautic is a cost-effective and customizable option that can handle growing subscribers. Being an open-source platform, it comes with a slightly steeper learning curve. However, since only a limited number of individuals will be using the tool, we can provide training to facilitate a smooth onboarding process. 
+
+## Comparison Matrix
+
+- ✅ Feature available, meets requirement
+- ❌ Feature not available, does not meet requirement
+- 🔄 Partial feature, limited feature availability, feature in progress or undergoing improvements
+- 1-3 Strength level
+- ❓Unknown
+
+| Factor                         | Sendy | Salesforce | MailChimp | Hubspot | Mautic | Adobe |
+| ---------------------------    | :---: | :--------: | :-------: | :-----: | :----: | :---: |
+| Usability                      |  2    |   1        |      3    |    3    |    1   |    3  |
+| Content Review                 |  🔄   |   ✅        |     ✅    |   ✅    |   ✅    |  ✅   |
+| Comments                       |   ❌  |   ✅        |     ❌    |   ✅    |   ✅    |   ✅  |
+| Open source*                   |  🔄   |    ❌       |     ❌    |   ❌    |   ✅    |   ❌  |
+| Multi-Media                    |   ✅  |   ✅        |     ❌    |   ✅    |   ✅    |   🔄  |
+| Analytics                      |    1  |   ✅        |      ✅   |    ✅   |   ✅    |   ✅  |
+| Internationalization           |     ❓ |   2        |      3    |    2   |    1   |   1   |
+| Onboarding costs               |     1 |   3        |      1    |    2    |    1    |    2 |
+| Community and Support*         |     1 |   3        |     3     |   3     |   3     |   3  |
+| Scalability                    |     🔄 |   ✅       |     ✅    |   ✅    |   ✅     | ❌    |
```

Comment:
Are the "nice to haves" the ones with the asterisks? if so, Scalability should have an asterisk as well. Maybe we move all the nice to haves to the bottom of the table.

Reply:
Yes, updated!